news/_posts/2019-09-11-prestashop-in-2019-and-beyond-part-2-pain-points.md

```diff
@@ -411,9 +411,9 @@ Some people insist that written specifications are not needed, because "the code
 
 Because once a bug appears, and specially when it's a consequence of two contradicting pieces of code, things get uncomfortable—if code _is_ the specification, which part is right and which part is a bug?
 
-In a project as complex as PrestaShop, it's vital that all expected behavior is well-documented. Otherwise, inconsistencies will keep creeping up again and again.
+In a project as complex as PrestaShop, it's vital that all expected behaviors are well-documented. Otherwise, inconsistencies will keep creeping up again and again.
```

Comment:
I think that Behavior is uncountable in this context: http://www.macmillandictionaryblog.com/language-tip-of-the-week-behaviou

Reply:
I had doubt about it, so I'd tend to agree with you. 
Again, something to ask a native, prolly :)

Reply:
```suggestion
In a project as complex as PrestaShop, it's vital that all expected behavior is well-documented. Otherwise, inconsistencies will keep creeping up again and again.
```

Reply:
I'm told the plural is right in this case.